nfadeeva/assignment4/tests.py

```diff
@@ -0,0 +1,34 @@
+from assignment4.count_islands import count_islands
+import unittest
+
+
+class Test(unittest.TestCase):
+    def test_example(self):
+        input_array = [[False, True, False, True],
+                       [True, True, False, False],
+                       [False, False, True, False],
+                       [False, False, True, False]]
+        num_rows = 4
+        num_cols = 4
+        self.assertEqual(count_islands(num_rows, num_cols, input_array), 3)
+
+    def test_empty_array(self):
+        input_array = []
+        num_rows = 0
```

Comment:
nit: you can also write:
```
num_rows, num_cols = 0, 0
```
This doesn't change anything, it's just more compact.